Makefile

```diff
@@ -156,6 +157,12 @@ test/common:
 	git checkout --force FETCH_HEAD -- test/common
 	git reset test/common
 
+SASS = ./node_modules/.bin/sass
+build-patternfly:
```

Comment:
This is a phony rule (i. e. not an actual file), and thus will break make dependencies. I. e. this will now run on each invocation of `make`, which will in turn require re-building the rpm and the VM (which is expensive), even if nothing changed. If you type `make` twice, the second one should be a no-op.

Let's instead make this a rule that specifies the actual inputs and outputs. Then you can also use the pre-defined make vars:

```make
dist/patternfly-4-base.css: $(srcdir)/lib/patternfly-4-cockpit.scss
	$(SASS) $< $@ --source-map ...
```